mllib/src/test/scala/org/apache/spark/mllib/clustering/LDASuite.scala

```diff
@@ -115,6 +115,15 @@ class LDASuite extends SparkFunSuite with MLlibTestSparkContext {
       assert(topicDistribution.toArray.sum ~== 1.0 absTol 1e-5)
     }
 
+    val top2TopicsPerDoc = model.topTopicsPerDocument(2).collect()
+    top2TopicsPerDoc.zip(topicDistributions).foreach {
+      case ((docId1, indices, weights), (docId2, topicDistribution)) =>
+        assert(indices.length == 2)
+        assert(weights.length == 2)
+        assert(docId1 == docId2)
+        assert(argtopk(topicDistribution.toBreeze, 2).toArray === indices)
+    }
```

Comment:
Shall we add one more `assert` to check the weights returned by `topTopicsPerDocument`?